linux-anvil-armv7l/Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM arm32v7/centos:7
+
+ADD qemu-arm-static /usr/bin/qemu-arm-static
+
+LABEL maintainer="conda-forge <conda-forge@googlegroups.com>"
+
+# Set an encoding to make things work smoothly.
+ENV LANG en_US.UTF-8
+ENV LANGUAGE=en_US.UTF-8
+
+# Add a timestamp for the build. Also, bust the cache.
+ADD http://worldclockapi.com/api/json/utc/now /opt/docker/etc/timestamp
+
+# configure yum and rpm for running on non-armv7l architectures
+RUN echo "armhfp" > /etc/yum/vars/basearch && \
+    echo "armv7hl" > /etc/yum/vars/arch && \
+    echo "armv7hl-redhat-linux-gpu" > /etc/rpm/platform
```

Comment:
`gpu` -> `gnu`?

Reply:
Yes, muscle memory must have let that sneak in.